airflow/contrib/operators/ssh_execute_operator.py

```diff
@@ -55,7 +55,7 @@ def __enter__(self):
         prefix = self._prefix
 
         pmktemp = ssh_hook.Popen(["-q",
-                                  "mktemp", "--tmpdir", prefix + "_XXXXXX"],
+                                  "mktemp", "-t", prefix + "_XXXXXX"],
```

Comment:
@bolkedebruin I've read the man page, according to my understanding, since we don't pass any argument to --tmpdir and don't set $TMPDIR, mktemp creates the file in /tmp, so it's equivalent to not passing --tmpdir, right?